src/backend/efiling-api/src/main/java/ca/bc/gov/open/jag/efilingapi/submission/SubmissionApiDelegateImpl.java

```diff
@@ -240,12 +237,12 @@ private UserDetails buildUserDetails(Submission submission) {
 
     @Override
     @RolesAllowed("efiling-user")
-    public ResponseEntity<SubmitFilingPackageResponse> submit(UUID xAuthUserId, UUID id, SubmitFilingPackageRequest submitFilingPackageRequest) {
+    public ResponseEntity<SubmitFilingPackageResponse> submit(UUID xAuthUserId, UUID submissionId, SubmitFilingPackageRequest submitFilingPackageRequest) {
         ResponseEntity response;
-        MDC.put(Keys.EFILING_SUBMISSION_ID, id.toString());
+        MDC.put(Keys.EFILING_SUBMISSION_ID, submissionId.toString());
         //TODO: this will get the submission details from the cache and build a submission object
         try {
-            SubmitFilingPackageResponse result = submissionService.submitFilingPackage(xAuthUserId, id, submitFilingPackageRequest);
+            SubmitFilingPackageResponse result = submissionService.submitFilingPackage(xAuthUserId, submissionId, submitFilingPackageRequest);
```

Comment:
Same as above: 
'xAuthUserId' doesn't need to be changed here to be 'xTransactionId'?